src/sensors/temperature/TMP36.cpp

```diff
@@ -0,0 +1,17 @@
+#include "TMP36.h"
+
+using namespace wlp;
+
+TMP36::TMP36(Board::AnalogPin analog_pin)
+    : pin(analog_pin) {}
+
+double TMP36::read_temperature() {
+    pin.powerup();
+    double reading = pin.sample();
+    double voltage = reading * 5.0;
+    voltage /= 1025.0;
```

Comment:
if you could make these constants `#define` at the top of this file it'd be better, names can be like `SCALE` or whatever